### PR TITLE
ringoromei changed to ringorseeber

### PR DESCRIPTION
**Pull request by vv-monsalve**
For consistency sake, the name was updated even in the pic to Ringo R Seeber (ringorseeber)

### CODE REVIEW

File: catalog/designers/ringorseeber/info.pb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+designer: "Ringo R. Seeber"
+link: "https://glyph.co"
+avatar: {
+  file_name: "ringoromei.png"
```
Comment (RosaWagner) [inline catalog/designers/ringorseeber/info.pb:4]: should be ringorseeber.png

**Comment by RosaWagner** [comment]
@m4rc1e can we remove "ringoromei" from the catalog and add "ringorseeber"? or we should just add "ringorseeber" and leave "ringoromei". 

It would not harm to remove it cause it is not used in any font. But want to be sure before merging

File: catalog/designers/ringorseeber/info.pb (hunk 1 above)
Comment (vv-monsalve) [reply catalog/designers/ringorseeber/info.pb]: Done. I forgot to commit that file

**Comment by RosaWagner** [comment]
Cool, so I can merge this and we'll see later if we can remove the doppelganger

**Pull request merged**